stl/src/xtime.cpp

```diff
@@ -87,9 +88,15 @@ _CRTIMP2_PURE long long __cdecl _Query_perf_counter() { // get current value of
 }
 
 _CRTIMP2_PURE long long __cdecl _Query_perf_frequency() { // get frequency of performance counter
-    LARGE_INTEGER li;
-    QueryPerformanceFrequency(&li); // always succeeds
-    return li.QuadPart;
+    static std::atomic<long long> freq_cached = 0;

```

Comment:
```suggestion
    static std::atomic<long long> freq_cached{0};
```
As of C++11, it was technically forbidden to copy-init `std::atomic`, because of its deleted copy constructor. With C++17's guaranteed copy elision, I no longer understand what the rules say, but I'd still like to see direct-init used for atomics.